VMBackup/main/handle.py

```diff
@@ -387,97 +392,131 @@ def daemon():
                 else:
                     backup_logger.log("the logs blob uri is not there, so do not upload log.")
                 backup_logger.log('commandToExecute is ' + commandToExecute, True)
+                
+                workload_patch = WorkloadPatch.WorkloadPatch(backup_logger)
+                
+                if workload_patch.name != None:
+                    backup_logger.log("workload backup enabled for workload: " + workload_patch.name, True)
+                    workload_patch.pre()
+                    if len(workload_patch.error_details) > 0:
+                        backup_logger.log("file system consistent backup only")
+                    #todo error handling
+                    if snapshot_type == "appAndFileSystem":
+                        g_fsfreeze_on = True
+                    else:
+                        g_fsfreeze_on = False
+                    freeze_snapshot(thread_timeout)
+                    workload_patch.post()
+                    workload_error = workload_patch.populateErrors()
+                    if workload_error != None and g_fsfreeze_on == False:
+                        run_status = 'error'
+                        run_result = workload_error.errorCode
+                        hutil.SetExtErrorCode(workload_error.errorCode)
```

Comment:
shouldnt it succed with FS consistent backup ?

Reply:
![image](https://user-images.githubusercontent.com/33341338/87034520-91810c00-c205-11ea-97dc-bd968fa0d8ad.png)

Reply:
tested the above flows